### PR TITLE
Add View Feature

### PR DESCRIPTION
**Pull request by dt-td**

### CODE REVIEW

File: src/main/java/seedu/address/ui/UiManager.java [hunk 1]
```diff
@@ -18,12 +18,12 @@
 public class UiManager implements Ui {
 
     public static final String ALERT_DIALOG_PANE_FIELD_ID = "alertDialogPane";
-
+    private static MainWindow mainWindow;
```
Comment (wpinrui) [inline src/main/java/seedu/address/ui/UiManager.java:21]: Seems like quite a big change to make this static, wonder if you could explain the rationale?

File: src/main/java/seedu/address/logic/commands/ViewCommand.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+
+/**
+ * Lists all persons in the address book to the user.
+ */
+public class ViewCommand extends Command {
+    public static final String COMMAND_WORD = "view";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Shows the person identified by the index number used in the displayed person list.\n"
```
Comment (Preshita01) [inline src/main/java/seedu/address/logic/commands/ViewCommand.java:22]: Maybe can replace 'person' to 'student' here?

File: src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.ViewCommand;
+
+public class ViewCommandParserTest {
```
Comment (Preshita01) [inline src/test/java/seedu/address/logic/parser/ViewCommandParserTest.java:12]: Maybe can just in case also include another test that tests for non-empty invalid inputs? (e.g. view a)

File: src/test/java/seedu/address/logic/commands/ViewCommandTest.java [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+
+        assertCommandSuccess(viewCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexFilteredList_throwsCommandException() {
+        showPersonAtIndex(model, INDEX_FIRST_PERSON);
+
+        Index outOfBoundIndex = INDEX_SECOND_PERSON;
+        // ensures that outOfBoundIndex is still in bounds of address book list
```
Comment (Preshita01) [inline src/test/java/seedu/address/logic/commands/ViewCommandTest.java:70]: Might be better to change 'address book list' to 'student list' so that it better fits TutorAid

File: README.md [hunk 5]
```diff
@@ -2,11 +2,10 @@
 [![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/AY2122S1-CS2103T-W16-3/tp/actions)
```
Comment (Preshita01) [inline README.md:2]: Next time it might be better to do documentation related work on another branch and add it in as a separate PR instead 😅
